js/src/personal/amanda/Amanda.module.css

```diff
@@ -1,14 +1,20 @@
 .landingPage {
```

Comment:
nit: Add linear to commit message.
nit: Commit message summary could be a bit shorter. Generally want it around 50 chars.